src/Integration.Vsix/CFamily/CLangAnalyzer.cs

```diff
@@ -34,9 +34,14 @@
 
 namespace SonarLint.VisualStudio.Integration.Vsix.CFamily
 {
+    internal interface ICLangAnalyzer : IAnalyzer
```

Comment:
To avoid confusion I think we shouldn't use `CLang` unless we are actually referring to something that is CLang-specific (and at some point we should rename `CLangAnalyzer`).
It might be better to call this `ICFamilyAnalyzer` (even though it means repeating the `CFamily` from the namespace).